rtil/src/native/character.rs

```diff
@@ -82,6 +83,11 @@ impl AMyCharacter {
     pub fn set_rotation(&mut self, pitch: f32, yaw: f32, roll: f32) {
         self.controller_mut().rotation = FRotator { pitch, yaw, roll };
     }
+    pub fn show_hud() {
+        if !AMYHUD.is_none() {
```

Comment:
Why is this check needed? `AMYHUD` should already be initialized before any rebo code is executed.

Reply:
Until the ordering is fixed, this will stay unresolved.